src/theme.ts

```diff
@@ -67,6 +88,13 @@ const theme: DefaultTheme = {
       color: ${({ theme }) => theme.colors.primary.default};
     }
   `,
+  devices: {
+    phone: 'only screen and (max-width: 576px)',
+    tabletPortrait: 'only screen and (min-width: 576px)',
+    tabletLandscape: 'only screen and (min-width: 768px)',
+    laptop: 'only screen and (min-width: 992px)',
+    laptopLarge: 'only screen and (min-width: 1200px)',
+  },
```

Comment:
Truly, I haven't understood why you need to handle if it is a Large Laptop, "medium" Laptop", a tablet, etc in so many moments. I mean, I understand that it is related to responsiveness, but it seamed like: "if there were so many moments in which you had to handle so many devices, wouldn't it be easier to remake the styles?"

Reply:
about having several devices, two comments:
- first of all, I opted to use Bootstrap breakpoints, which I think is a good "standard"/reference for media queries (if you check other design systems like material and antd you might find something similar);
- then I thought it'd be a good idea to give "meaningful" names for those breakpoints. (I could only think in terms of phones, tablets and laptops/desktop -- open to suggestions tho)

Remember that I'm setting this values up in the theme, so I can use wherever I want with such name, instead of manually specifying the media query in every occasion. This way I can code thinking "I want this to look like that in a phone", and not "I want this to look like that in a screen with up to 576 pixels" (in mind it's easier).